<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>502</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>R2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="scrollableRegion" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>913</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>911</width>
       <height>191</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Timestamp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Wait (msec)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Verify</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>913</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Control</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>211</width>
         <height>123</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>24V CONTROL BOX</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="b2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>26</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>W1</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>26</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>W2</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>26</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>W3</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y1</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>59</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y2</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b7">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>59</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>O</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>92</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>G</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b9">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>92</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BK</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="b10">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>92</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
       <zorder>b2</zorder>
       <zorder>b3</zorder>
       <zorder>b4</zorder>
       <zorder>b5</zorder>
       <zorder>b6</zorder>
       <zorder>b7</zorder>
       <zorder>b8</zorder>
       <zorder>b9</zorder>
       <zorder>b10</zorder>
       <zorder>groupBox_5</zorder>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>10</y>
         <width>351</width>
         <height>58</height>
        </rect>
       </property>
       <property name="title">
        <string>VERIFY</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>20</y>
          <width>326</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>76</y>
         <width>351</width>
         <height>45</height>
        </rect>
       </property>
       <property name="title">
        <string>RESULT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>18</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>PASS</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>18</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>FAIL</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>18</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>INVALID</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>130</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>COMMENT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>20</y>
          <width>326</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>TEST BROWSER</string>
        </property>
       </column>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>14</number>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>MODE</string>
       </property>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>23</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>77</x>
          <y>23</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>151</x>
          <y>20</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SEND</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Test</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Script"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
   <addaction name="menuRecord"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Test Case</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionReset_Script">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/clean.png</normaloff>:/images/clean.png</iconset>
   </property>
   <property name="text">
    <string>Reset Script</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
